### PR TITLE
Add abstraction for Custom Tokenizer

### PR DESCRIPTION
**Pull request by JRC1995**
Related to https://github.com/vllm-project/vllm/issues/111

```
model = LLM(abcd)
model.set_tokenizer(custom_tokenizer)
```

### CODE REVIEW

File: vllm/entrypoints/llm.py [hunk 1]
```diff
@@ -60,6 +60,11 @@ def get_tokenizer(
     ) -> Union[PreTrainedTokenizer, PreTrainedTokenizerFast]:
         return self.llm_engine.tokenizer
 
+    def set_tokenizer(
+        self, tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast]
+    ):
```
Comment (WoosukKwon) [inline vllm/entrypoints/llm.py:65]: style nit:
```suggestion
        self,
        tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast],
    ) -> None:
```

**Review by WoosukKwon — COMMENTED** [review]
Thanks for the fix! I was actually working on this now. 😄 Please check a minor comment.

**Comment by JRC1995** [comment]
Thanks. I committed the typing.

**Review by WoosukKwon — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**